### PR TITLE
Update showcase.json

### PR DESCRIPTION
**Pull request by jdchibuk**
re: "user controlled mind and body canister”

Thank you for your contribution to the IC Developer Portal.
Before submitting your Pull Request, please make sure that:

- [ ] You have updated the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file according to your changes.
- [ ] You have registered new documents to [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js) if any.

### CODE REVIEW

**Comment by sa-github-api** [comment]
Dear @jdchibuk,

In order to potentially merge your code in this open-source repository and therefore proceed with your contribution, we need to have your approval on DFINITY's [CLA](https://github.com/dfinity/cla/blob/main/CLA.md).

If you decide to agree with it, please visit this [issue](https://github.com/dfinity/cla/issues/424) and read the instructions there. Once you have signed it, re-trigger the workflow on this PR to see if your code can be merged.

— The DFINITY Foundation

File: showcase.json [hunk 1]
```diff
@@ -3245,5 +3245,17 @@
     ],
     "usesInternetIdentity": true,
     "github": "https://github.com/id-daniel-mccoy/asset-app"
-  }
-]
\ No newline at end of file
+  },{
+  "name": "user controlled mind and body canister”,
```
Comment (jennifertrin) [inline showcase.json:3249]: Can you please add an `id` key-value pair?

File: showcase.json [hunk 2]
```diff
@@ -3245,5 +3245,17 @@
...
+  "website": "https://github.com/blueberryxtech/ic-mind-body-canister”,
+  "oneLiner": "tooling for uploading and accessing mind and body data from a firebase API onto ICP canister, blueberry data as a reference example",
+  "tags": [
+      "Tools / Infrastructure"
+    ],
+  "description": "POC of a decentralized mind and body data management tooling that leverages Internet Computer (IC) technology. This POC provides methods and interface to submit HTTP Outcall requests to a proxy API that interfaces with a Firebase FIrestore database. It provides an example front end react application that interacts with a motoko backend.”,
+  "stats": "100% on IC mainnet”,
```
Comment (jennifertrin) [inline showcase.json:3256]: Can you please remove this as a stat? We normally suggest a stat to list the number of users, etc.

File: showcase.json [hunk 3]
```diff
@@ -3245,5 +3245,17 @@
...
+  "usesInternetIdentity": true,
+  "github": "https://github.com/blueberryxtech/ic-mind-body-canister”,
+  "submittableId": "44190088"
```
Comment (jennifertrin) [inline showcase.json:3259]: Can you please include a logo as well? Please see logo key-value pair in other projects as an example and upload a logo to the correct file path.

**Review by jennifertrin — CHANGES_REQUESTED** [review]
Can you please also review the CLA steps?

**Comment by jdchibuk** [comment]
should be updated for format now

here is CLA confirmation: https://github.com/dfinity/cla/issues/424#issuecomment-1883433708

**Comment by jennifertrin** [comment]
> should be updated for format now
> 
> here is CLA confirmation: [dfinity/cla#424 (comment)](https://github.com/dfinity/cla/issues/424#issuecomment-1883433708)

Thank you. Can you add a file to the repo instead of calling a link for the logo?

**Comment by jdchibuk** [comment]
got it - added to img/showcase/ folder - let me know if this works - appreciate the help!

**Comment by jennifertrin** [comment]
> should be updated for format now
> 
> here is CLA confirmation: [dfinity/cla#424 (comment)](https://github.com/dfinity/cla/issues/424#issuecomment-1883433708)



> got it - added to img/showcase/ folder - let me know if this works - appreciate the help!

Yes, this works! Thank you!

**Review by jennifertrin — APPROVED** [review]

**Pull request merged**